### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by konichi3**

### CODE REVIEW

**Review by marianan — APPROVED** [review]
:shipit:

File: README.md [hunk 1]
```diff
@@ -1,6 +1,76 @@
...
+&& sudo -H ./azure-iot-edge-installer.sh -s <IDScope> -r <RegistrationID> -k <Symmetric Key> \
+&& rm -rf azure-iot-edge-installer.sh
+
+```
+Upon completion of the script, go to All devices view and Device status should be changed from registered to provisioned.
+
+### Uninstalling IoT Edge runtime and Moby-Engine
+You can uninstall both IoT Edge runtime and Moby-Engine with following commands
+
+- Uninstall Edge runtime
```
Comment (mhshami01) [inline README.md:57]: These steps should be run in sequence so we should consider enumerating them (1., 2., etc)

**Review by mhshami01 — APPROVED** [review]

**Review by cindydeng1998 — APPROVED** [review]
🌞

File: README.md (hunk 1 above)
Comment (konichi3) [reply README.md]: Thanks. I changed to the numbers instead of bullets

**Pull request merged**